File: Front End/src/service/patient.ts
import { Cause } from "./cause";

export interface Patient {
  id: number;
  profilePicture: string;
  firstName: string;
  lastName: string;
  age: number;
  gender: string;
  phoneNumber: string;
  diagnosis: string;
  showButtons: boolean;
  dropdownOption?: string;
  causes?: Cause[];
  selectedOption?: string;
  fontColor?: string;
  showOverlay?: boolean;
}

File: Back End/BigBang Assessment 2/Models/Patient.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BigBang_Assessment_2.Models
{
    public class Patient
    {
        [Key]
        public int Id { get; set; }

        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public int? Age { get; set; }
        public string? Gender { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string? Address { get; set; }

    }
}
